projects/plugins/crm/includes/jpcrm-mail-templating.php

```diff
@@ -531,6 +530,11 @@ function zeroBSCRM_quote_generateAcceptNotifHTML( $quoteID = -1, $quoteSignedBy
 				$zbs_biz_youremail = zeroBSCRM_getSetting( 'businessyouremail' );
 				$zbs_biz_yoururl = zeroBSCRM_getSetting( 'businessyoururl' );
 
+				$quote_url                      = zeroBSCRM_portal_linkObj( $quoteID, ZBS_TYPE_QUOTE ); // phpcs:ignore WordPress.NamingConventions.ValidVariableName.VariableNotSnakeCase
+				$quote_edit_url                 = jpcrm_esc_link( 'edit', $quoteID, 'zerobs_quote' ); // phpcs:ignore WordPress.NamingConventions.ValidVariableName.VariableNotSnakeCase
+				$replacements['quote-url']      = $quote_url;
+				$replacements['quote-edit-url'] = $quote_edit_url;
```

Comment:
Could collapse into two lines if you wanted; not a blocker.

Reply:
Also, maybe we should move these further down to where we set the other `$replacements` keys.

Reply:
Yes, good idea, added in d3edfc15bae81dc48adbb5cf67604636df3cc58e